src/generated/virgl_renderer_bindings.rs

```diff
@@ -48,6 +48,7 @@ pub const VIRGL_RENDERER_MAP_CACHE_WC: u32 = 3;
 pub const VIRGL_RENDERER_BLOB_FD_TYPE_DMABUF: u32 = 1;
 pub const VIRGL_RENDERER_BLOB_FD_TYPE_OPAQUE: u32 = 2;
 pub const VIRGL_RENDERER_BLOB_FD_TYPE_SHM: u32 = 3;
+pub const VIRGL_RENDERER_BLOB_FD_TYPE_APPLE: u32 = 4;
```

Comment:
`VIRGL_RENDERER_BLOB_FD_TYPE_APPLE` is not available in upstream virglrenderer.  I would think we would want to be more specific anyways (`HANDLE_TYPEMTLTEXTURE`) when we get to it.  Maybe you can just use `cfg(target_os = 'macos')` to differentiate apple fds/Linux fds: but opaque fd should work with both too.